src/evict/evict_inline.h

```diff
@@ -646,9 +646,11 @@ __wt_evict_app_assist_worker_check(
         __wt_atomic_loadv64(&txn_global->current) != __wt_atomic_loadv64(&txn_global->oldest_id));
 
     /*
-     * LSM sets the "ignore cache size" flag when holding the LSM tree lock, in that case, or when
-     * holding the handle list, schema or table locks (which can block checkpoints and eviction),
-     * don't block the thread for eviction.
+     * Don't block the thread for eviction when holding the handle list, schema or table locks
```

Comment:
Is the added detail intentional? If so, great. Also "session's" -> "sessions", I think?

Reply:
Yes, this is intentional

>"session's" -> "sessions"
yes you're right

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/11273/commits/948ac1fd23f1048fc68bf8c464fb6babe5b7a8bb